admin/class-admin-apple-meta-boxes.php

```diff
@@ -107,13 +107,25 @@ public function do_publish( $post_id, $post ) {
 	 * @static
 	 */
 	public static function save_post_meta( $post_id ) {
-		// Save fields from the meta box
-		if ( ! empty( $_POST['apple_news_sections'] ) ) {
-			$sections = array_map( 'sanitize_text_field', $_POST['apple_news_sections'] );
+
+		// Save section override setting.
+		$mappings = get_option( Admin_Apple_Sections::TAXONOMY_MAPPING_KEY );
```

Comment:
The other class constants are all uppercase. I can standardize the ones you added in the themes branch to uppercase to match.